### DISCUSSION TITLE
How to Turn off User Feedback form?

### DISCUSSION THREAD

**Discussion by dipakparmar** [stayallive/wp-sentry#123] [Q&A] [answered]
Is there an env variable or define config that can be passed to turn off the user feedback form?

**Comment by stayallive** [comment]
Hey @dipakparmar, where are you seeing the feedback form? 

The thing is not enabled by default and there is also no way to set a config option to have it popup in this plugin so I'm curious why you are asking this :)

**Reply by dipakparmar** [reply]
Hi @stayallive thanks for getting back to me. The form I get is when there is any crash.

For Reference: https://docs.sentry.io/platforms/javascript/enriching-events/user-feedback/

**Reply by stayallive** [reply]
I know of which form you are speaking. But what I meant is that this is not a feature that can be enabled by default so I'm puzzled why you are seeing it. Are you 100% sure it's caused by this exact plugin? Or are you maybe using another Sentry plugin or some custom code calling `Sentry.showReportDialog()`?

**Reply by dipakparmar** [reply]
Found out there was a javascript snippet that was passed in a custom theme which was prompting the feedback form. Thanks for all the help and quick replies. 😄

**Comment by dipakparmar** [comment, marked as answer]
Answer for anybody else also experience a similar issue, check the javascript trace which plugin/theme adding it to front-end.

**Reply by uhlhosting** [reply]
@dipakparmar and in the end did you managed to get it working?

**Reply by dipakparmar** [reply]
> @dipakparmar and in the end did you managed to get it working?

Yes!

**Reply by uhlhosting** [reply]
@dipakparmar how you did it?

**Reply by dipakparmar** [reply]
@uhlhosting In my case, another third-party theme had the function invoking sentry report dialogue; I was able to find it using browser developer tools. and not part of this plugin at the time of this discussion.
